### DISCUSSION TITLE
Leave Blank v21.02.2

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#26] [General]
I always use your builts on the nanopi r2s. Last version is installed but when I do a speedtest, the internet speed goes from max speed to 0 Mbps.
Than the connection is lost as well.
Didn't had this in the previous built. 
What had been changed there?

Edit: went back to the  v21.02.1 version and this one works directory without this issue.

**Comment by tommiefr** [comment]
Any news about this issue?
Or are you releasing the 21.02.3 version soon?

https://forum.openwrt.org/t/openwrt-21-02-3-third-service-release/125732

**Reply by nicholas-opensource** [reply]
https://github.com/nicholas-opensource/OpenWrt-Autobuild/releases/download/1.3/R2S_v21.02.3_0430.zip
Sorry for the delayed releases

**Comment by nicholas-opensource** [comment]
Hi tommiefr, are there more detailed system logs? Or compare the `Firewall Status` between different versions

**Reply by tommiefr** [reply]
No sorry, i don't have the log for you. I already went back to the previous built.
For now i will wait for the 22.03 version, Just saw the rc1 was released earlier.

**Reply by nicholas-opensource** [reply]
> No sorry, i don't have the log for you. I already went back to the previous built.
> For now i will wait for the 22.03 version, Just saw the rc1 was released earlier.

Testing everything for v22.03, but unfortunately can't use firewall4, nftables has poor compatibility. iptables-nft doesn't work fine.
If all goes well, the v22.03.0-rc1 will be pushed soon

**Reply by tommiefr** [reply]
Any news about the 22.03.0 RC version?
